.env

```diff
@@ -1,5 +1,14 @@
+# SERVER
 SERVER_PORT=3000
-INFLUX_PORT=8086
+
+# DOCKER
 WORKDIR=/usr/src/app/
+
+# STORAGE
 FILES_LOCATION=./data
-STORAGE=files
+
+# INFLUX
+# Set to `aggr-influx` if you use docker
+# "Config.json property influxHost must have the same value"
+# INFLUX_HOST=aggr-influx
+# INFLUX_PORT=8086
```

Comment:
Seems like I need to uncomment the INFLUX_PORT variable to make it work
<img width="596" alt="image" src="https://user-images.githubusercontent.com/5738336/224393232-13babe47-2f84-40d2-9ffd-cf57052a177c.png">

I'm confused because if I don't want to use influx, the `docker-compose build` is still asking for that variable